README.md

```diff
@@ -4,3 +4,35 @@ This repository contains ports of Tock OS (https://www.tockos.org) to Titan
 chips.
 
 This is not an officially supported Google product.
+
+
+## Getting started
+
+### Clone the repo
+
+Get the source:
+
+```shell
+git clone --recursive git@github.com:google/tock-on-titan.git
+```
+
+### Get the tools and libs to build the code
+
+Download Rust
+
+```shell
+cd tock-on-titan
+curl https://sh.rustup.rs -sSf | sh
+```
+
+Configure Rust
+
+```shell
+make setup
```

Comment:
Oh, I didn't realize we had that, cool! Does that make the manual "rustup" calls obsolete? Can we test that by purging some rust directory?

Reply:
Don't think so.  `make setup` does exactly what the explicit `rustup` commands did.